wagtail/models/__init__.py

```diff
@@ -1314,7 +1314,7 @@ def get_specific(self, deferred=False, copy_attrs=None, copy_attrs_exclude=None)
             return self
 
         if isinstance(self, model_class):
-            # self is already the an instance of the most specific class
+            # self is already an instance of the most specific class.
             return self
```

Comment:
I think this change here was made by accident

Reply:
Yes, thanks for mentioning that!